code/game/machinery/doors/airlock.dm

```diff
@@ -650,11 +651,11 @@ About the new airlock wires panel:
 /obj/machinery/door/airlock/Topic(href, href_list, var/nowindow = 0)
 	// If you add an if(..()) check you must first remove the var/nowindow parameter.
 	// Otherwise it will runtime with this kind of error: null.Topic()
-	if(!isAI(usr) && usr.z != z) return 1
+	if(!isAI(usr) && !istype(usr,/mob/living/silicon/pai) && usr.z != z) return 1

```

Comment:
The only way it can work. The actual pAI mob itself is on z = 0 for some strange reason.